### PR TITLE
build(deps): update to docusuarus 2.0.0-alpha.59

### LINKED ISSUE #9: docs: check for broken links
**Description**

Add a CI job to check for broken links in the documentation.

Could look into using something like [broken-link-checker](https://www.npmjs.com/package/broken-link-checker) or [linkinator](https://github.com/JustinBeckwith/linkinator) from the generated preview site.

Complexity [S/M/L]: S

### PR DESCRIPTION
**Pull request by danielhochman**
### Description
Update docusaurus.

### Testing Performed
CI

fixes #9

### CODE REVIEW

File: docs/_website/docusaurus.config.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ module.exports = {
     'https://fonts.googleapis.com/css2?family=Roboto&family=Open+Sans&display=swap',
     'https://cdn.rawgit.com/luizbills/feather-icon-font/v4.7.0/dist/feather.css',
   ],
-  plugins: ['@docusaurus/plugin-google-analytics'],
+  plugins: [],
```
Comment (danielhochman) [inline docs/_website/docusaurus.config.js:13]: confirmed that GA is still included in page. the build was throwing an error about duplicate plugins before this change. guessing it's auto-included from the core theme.

Comment (dschaller) [reply docs/_website/docusaurus.config.js:13]: huh that's awesome. They include the package as a dep in core so this makes sense.

File: docs/_website/src/theme/Navbar/index.js [hunk 2]
```diff
@@ -14,6 +14,28 @@ import useLogo from '@theme/hooks/useLogo';
 
 import styles from './styles.module.css';
 
+// items defined here instead of config so they can have an associated icon
+var items = [
```
Comment (dschaller) [inline docs/_website/src/theme/Navbar/index.js:18]: It looks like the navbar is now missing the docs, github, and community links

Comment (danielhochman) [reply docs/_website/src/theme/Navbar/index.js:21]: fixed, ptal

**Pull request merged**